core-server/src/main/java/org/orbisgis/orbisserver/control/xml/GetCapabilitiesController.java

```diff
@@ -0,0 +1,127 @@
+ *
+ * The GIS group of the DECIDE team is located at :
+ *
+ * Laboratoire Lab-STICC – CNRS UMR 6285
+ * Equipe DECIDE
+ * UNIVERSITÉ DE BRETAGNE-SUD
+ * Institut Universitaire de Technologie de Vannes
+ * 8, Rue Montaigne - BP 561 56017 Vannes Cedex
+ *
+ * OrbisServer is distributed under LGPL 3 license.
+ *
+ * Copyright (C) 2007-2014 CNRS (IRSTV FR CNRS 2488)
+ * Copyright (C) 2015-2017 CNRS (Lab-STICC UMR CNRS 6285)
+ *
+ * This file is part of OrbisGIS.
+ *
+ * OrbisServer is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * OrbisServer is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with
+ * OrbisServer. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For more information, please consult: <http://www.orbisgis.org/>
+ * or contact directly:
+ * info_at_ orbisgis.org
+ */
+
+package org.orbisgis.orbisserver.control.xml;
+
+
+ import org.orbiswps.server.WpsServer;
+ import org.orbiswps.server.model.JaxbContainer;
+ import org.wisdom.api.DefaultController;
+ import org.wisdom.api.annotations.Controller;
+ import org.wisdom.api.annotations.Parameter;
+ import org.wisdom.api.annotations.Route;
+ import org.wisdom.api.annotations.View;
+ import org.wisdom.api.http.HttpMethod;
+ import org.wisdom.api.http.Result;
+ import org.wisdom.api.templates.Template;
+ import org.slf4j.Logger;
+ import org.slf4j.LoggerFactory;
+ import org.xnap.commons.i18n.*;
+
+ import org.orbisgis.orbisserver.control.web.IndexController;
+
+ import javax.xml.bind.JAXBElement;
+ import javax.xml.bind.JAXBException;
+ import javax.xml.bind.Marshaller;
+ import javax.xml.bind.Unmarshaller;
+
+ /**
+  * Instance of DefaultController containing a WpsServer.
+  *
+  * @author Guillaume MANDE
+  */
+ @Controller
+ public class GetCapabilitiesController extends DefaultController {
+
+   /**
+    * IndexController's object, used to get some useful attributes.
+    */
+   private IndexController indexController = new IndexController();
+
+   /** Logger */
+   private static final Logger LOGGER = LoggerFactory.getLogger(GetCapabilitiesController.class);
+   /** I18N object */
+   private static final I18n I18N = I18nFactory.getI18n(IndexController.class);
+
+   /**
+    * The action method returning the xml file corresponding to the GetCapabilities method. It handles
+    * HTTP GET request on the "/orbisserv/ows" URL. 
+    * A good request should be http://localhost:9000/orbisserver/ows?service=WPS&version=2.0.0&request=GetCapabilities
+    *
+    * @Parameter the service
```

Comment:
The javadoc parameter tag work this way : `@param theParameterName The description of the parameter.`